### PR TITLE
Security Webbing

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR sees the addition of an expensive item to the Security vendor, the security webbing. A unique chest rig.

The concept of this item is to give Officers something to spend their money on when they save up. 

webbing

The chest rig is intended as an expensive minor upgrade over the standard security belt that security personal can purchase. Functionally it is identical to the security belt except it can hold one extra item, up to 6 from 5.

The belt costs $800, meaning round start Officers will be unable to afford it. The sec vendor holds five of these webbings.

:cl: Steelpoint
add: Nanotrasen Security has unveiled a newly designed Security Webbing, a minor upgrade over the standard issue security belt that is comfortable, tactical and able to hold an extra security item. 
add: However, the webbing is not considered standard issue, so any interested officers will need to buy the item from a SecVendor out of their own pay.
/:cl:

### CODE REVIEW

**Comment by Borisvanmemes** [comment]
Is this the part where everyone screams at steelpoint for  buffing sec?
I feel like this is the point where everyone screams at steelpoint for buffing sec.

**Comment by Steelpoint** [comment]
Back when the new economy system was implemented a lot of discussions were made on the concept of allowing crew to purchase expensive items that are a very minor upgrade over existing items. I was inspired by this when I late read the big discussions, and I really like the webbing sprites and thus this PR was born.

**Comment by Mickyan** [comment]
you can't figure out how to customize the cost of items because custom_price is bugged for premium items

**Comment by Steelpoint** [comment]
Good to know.

**Comment by Dorsisdwarf** [comment]
Sec buff from steelpoint ASIDE theres nothing wrong with this. 👍

**Comment by FantasticFwoosh** [comment]
Time to freeze some criminal accounts if the Warden/HOS doesn't get mugged by powergaming officers to pay off these bad boys.

@Steelpoint did you check that without access (because its ID access determined) everything costs properly not $800 for other things. By spawning in as a assistant(/other) on self-hosted private game and either spawning a secvendor onto yourself or the means to break in.

- Still could be a useful for traitors who take the briefcase of money to simply buy out premium gear from department vendors like this.

**Comment by Steelpoint** [comment]
@FantasticFwoosh I already tested and the cost remains at $800 for the webbing as an Assistant. Every other items retains their original cost of $100 (cuffs, flashbang, etc).

**Comment by Steelpoint** [comment]
One thing I welcome is suggestions on the pricing of the webbing. The $800 is more a placeholder.

For reference, roundstart Officers start with $600. I do not know how much they earn every payminute.

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -214,6 +214,38 @@
...
+		/obj/item/ammo_casing/shotgun,
+		/obj/item/ammo_box,
+		/obj/item/reagent_containers/food/snacks/donut,
+		/obj/item/kitchen/knife/combat,
+		/obj/item/flashlight/seclite,
+		/obj/item/melee/classic_baton/telescopic,
+		/obj/item/radio,
+		/obj/item/clothing/gloves,
+		/obj/item/restraints/legcuffs/bola
+		))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/storage/belt.dm]: This can either pull from inheritance or can pull from one of the global lists.

Comment (Steelpoint) [reply code/game/objects/items/storage/belt.dm]: Good point.

File: code/modules/vending/security.dm [hunk 2]
```diff
@@ -15,11 +15,12 @@
...
 	contraband = list(/obj/item/clothing/glasses/sunglasses = 2,
 		              /obj/item/storage/fancy/donut_box = 2)
-	premium = list(/obj/item/coin/antagtoken = 1)
+	premium = list(/obj/item/storage/belt/security/webbing = 5,
+					/obj/item/coin/antagtoken = 1)
 	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
 	resistance_flags = FIRE_PROOF
 	default_price = 100
-	extra_price = 150
+	extra_price = 800
```
Comment (ShizCalev) [inline code/modules/vending/security.dm]: Fix so you can revert this; #41399

**Comment by Shadowflame909** [comment]
It's not that big of a buff and ultimately doesn't have much effect on the game other then you now have to dodge 3 flash bangs being thrown at you instead of 2.

**Comment by Daxxed** [comment]
Biased miner main loves his webbings, does this come with extra protection aswell ?
Either way I'd love this.

**Comment by improvedname** [comment]
this just ends in. hos gives officers money to buy one from his mediocre useless budget ID

File: code/modules/vending/security.dm (hunk 2 above)
Comment (Steelpoint) [reply code/modules/vending/security.dm]: I'll remedy this.

**Comment by Iamgoofball** [comment]
800 for webbing is good.

**Comment by Anonmare** [comment]
Does the sprite update with the baton/flashbang/etc as the sec belt does?

**Comment by Steelpoint** [comment]
No it does not, I disabled that function for the webbing.

**Comment by optimumtact** [comment]
This is excellent

**Comment by ShizCalev** [comment]
@steelpoint

Oh btw pricing for premium items is based off the custom_premium_price var.

So these are live at $150 spacebucks ATM :^)

**Comment by Steelpoint** [comment]
The PR that fixes that was merged just after this one, so hopefully that should resolve it.

**Pull request merged**